app/src/main/java/com/framgia/englishconversation/screen/createPost/CreatePostViewModel.java

```diff
@@ -509,12 +518,15 @@ private void onAudioRecordingClick() {
                 new RecordingAudioDialog.OnRecordingAudioListener() {
                     @Override
                     public void onRecordingAudioClick(String filePath, String fileName) {
+                        releaseExoAudioPlayer();
+                        Log.d("CratePostViewModel", filePath);
                         MediaModel record = new MediaModel(MediaModel.MediaType.AUDIO);
                         record.setId(UUID.randomUUID().toString());
                         record.setUrl(filePath);
                         record.setName(fileName);
                         initMedia(filePath);
                         addPostAudioRecord(record);
+                        initializeAudioPlayer();
```

Comment:
ERROR: 'method def' child have incorrect indentation level 24, expected level should be one of the following: 16, 20.